### ISSUE TITLE
Regression on parameter when using them in data rows

### ISSUE THREAD

**Issue by patrick-blanc** [SharePoint/PnP-Sites-Core#2263]
#### Category
[x] Bug only with version 3.9.1905 or higher. Working fine with previous version.

#### Environment
[x] Office 365 / SharePoint Online

#### Expected or Desired Behavior
I have a provisioning template with some paramters. 
```xml
<pnp:Parameters>
  <pnp:Parameter Key="test"></pnp:Parameter>
</pnp:Parameters>
```

I used those parameters inside datarow datavalue
```xml
<pnp:DataRows KeyColumn="Key" UpdateBehavior="Overwrite">
  <pnp:DataRow>
    <pnp:DataValue FieldName="Key" >test</pnp:DataValue>
    <pnp:DataValue FieldName="Value">{parameter:test}</pnp:DataValue>
  </pnp:DataRow>
</pnp:DataRows>
```

When settings parameters in c# on the template or in cmdlet in powershell the list item is created successfully but the value inside it should be the value of the parameter.

#### Observed Behavior
The value of the list item column contains {parameter:test} instead of the value set when applying the template.

#### Steps to Reproduce

- Install module 3.9.1905 or higher.
- Add parameter to a pnp template.
- User the parameter in data row data value
- Apply the template

**Comment by chrriedel** [comment]
same here. {parameter:Value} is currently not parsed as a token. Instead the string is inserted into the Datarow.
PnP Version 3.9.1905.3

**Comment by gszdev** [comment]
Maybe have a look at the following pull request:
Fix DataRow parameters replacement #2248

see also #2258, #2259

@jansenbe @erwinvanhunen 
Is it possible to merge PR #2248 or implement another fix for Data Rows?

**Comment by jansenbe** [comment]
Fix has been released as part of the June release
